src/frontend/citizen-portal/src/app/components/find-dispute/find-dispute.component.ts

```diff
@@ -52,6 +52,7 @@ export class FindDisputeComponent implements OnInit {
     if (!validity) {
       return;
     }
-    this.busy = this.violationTicketService.searchTicket(this.form.value).subscribe(res => res);
```

Comment:
review the warnings given below

Reply:
this line is just the copied line from the search ticket page, changed to search dispute